src/fare-contracts/details/OrderDetails.tsx

```diff
@@ -15,6 +16,11 @@ export const OrderDetails = ({fareContract}: {fareContract: FareContract}) => {
     FareContractTexts.details.orderId(fareContract.orderId),
   );
   const firstTravelRight = fareContract.travelRights[0];
+  const priceString =
+    fareContract.totalAmount && fareContract.totalAmount !== ''
```

Comment:
Since the empty string is falsy, the last check is redundant here

```suggestion
    fareContract.totalAmount
```